src/main/java/dasniko/testcontainers/keycloak/KeycloakContainer.java

```diff
@@ -20,17 +20,27 @@
 import org.jboss.shrinkwrap.api.exporter.ZipExporter;
 import org.jboss.shrinkwrap.api.importer.ExplodedImporter;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jetbrains.annotations.Nullable;
```

Comment:
I know that these checks might be good and important. But for now I haven't used them yet in this project and I rather would introduce them more holistic than with single PRs. Please remove this annotation. If you feel this project should use them more widely, open an issue for this. Thanks.